src/cli/node/json_rpc_client.rs

```diff
@@ -551,24 +554,42 @@ pub fn send_vtt(
     pkh: Option<PublicKeyHash>,
     value: u64,
     size: Option<u64>,
-    fee: u64,
+    fee: Option<u64>,
+    suggest_fee: bool,
     time_lock: u64,
     sorted_bigger: Option<bool>,
     dry_run: bool,
 ) -> Result<(), failure::Error> {
     let mut stream = start_client(addr)?;
+    let mut id = 1;
 
     let size = size.unwrap_or(value);
     if value / size > 1000 {
         bail!("This transaction is creating more than 1000 outputs and may not be accepted by the miners");
     }
 
+    // Prepare for fee estimation if enabled through `--suggest-fee` flag.
+    let (fee, estimate) = match (suggest_fee, fee) {
+        (true, _) | (_, None) => {
```

Comment:
I would prefer an error message when the user does not specify neither `--suggest-fee` nor a `--fee`. Because `--suggest-fee` waits for user input, so not sure if it is a good default. Also some way to make `suggest-fee` non-interactive would probably be useful.

```suggestion
        (true, _) => {
```

Reply:
> Also some way to make `suggest-fee` non-interactive would probably be useful.

Sir, the non-interactive version of `--suggest-fee` is `priority` plus `--fee`.

Reply:
> I would prefer an error message when the user does not specify neither `--suggest-fee` nor a `--fee`. Because `--suggest-fee` waits for user input, so not sure if it is a good default. 

Funnily enough, I had gone the other direction and dropped the `--suggest-fee` flag and rather bring that feature if no `--fee` flag is provided, given that it is explicitly defined and documented as optional.

Reply:
> Funnily enough, I had gone the other direction and dropped the --suggest-fee flag and rather bring that feature if no --fee flag is provided, given that it is explicitly defined and documented as optional.

That also makes sense.

The problem with trying to use the `priority` command is that `send --fee` expects an absolute fee, so users would need to manually calculate the weight of the transaction. I guess the logical solution is to implement a `send --priority`.